overlord/devicestate/handlers_install.go

```diff
@@ -341,6 +341,45 @@ func (m *DeviceManager) doSetupRunSystem(t *state.Task, _ *tomb.Tomb) error {
 		return fmt.Errorf("cannot make system runnable: %v", err)
 	}
 
+	// TODO: FIXME: this should go away after we have time to design a proper
+	//              solution
+	// TODO: only run on specific models?
```

Comment:
@mvo5 what do you think about this? should we only run this on the specific model or is it okay to run all the time?

Reply:
ack, then I'll force merge the PR if we decide to move forward with this tomorrow then